app/Process.php

```diff
@@ -2,13 +2,36 @@
 
 namespace App;
 
+use Illuminate\Support\Carbon;
+
 class Process
 {
     protected $commands = [];
     protected $cwd = null;
     protected $asUser = null;
     protected $output = true;
 
+    /** @var resource $outputFile */
+    protected $outputFile;
+
+    public function __construct()
+    {
+        $filename = rtrim(sys_get_temp_dir() . '/fwd_output_' . Carbon::now()->format('Ymdhis'));
+        $this->outputFile = fopen($filename, 'w+') ?: fopen('/dev/null', 'w+');
```

Comment:
We cannot open a single file and use it for all runSilently commands... I think we should move this to a `prepareOutputBuffer()` method that is called every time we call runSilently, maybe even from the `getDescriptors()` method...

Reply:
We can set the file before running silently a command and unset it after that. This way, we won't open a file every time we instantiate this class, only when we need it. I have only one question, do we need to keep a outputs stack? Or just the last output matters for us?

Reply:
I guess we only care for the last one.